sql/pr/inactivity-on-prs.sql

```diff
@@ -0,0 +1,110 @@
+-- Time PRs have gone without any updates
+WITH
+    pulls AS (
+        SELECT
+            owner,
+            repo,
+            number AS pull_number,
+            created_at,
+            issue_url
+        FROM
+            unique_pulls
+        WHERE
+                owner = 'trinodb'
+          AND repo = 'trino'
+          AND state = 'open'
+    ),
+    last_review_comments AS (
+        SELECT
+            r.pull_number,
+            CAST (CURRENT_TIMESTAMP AT TIME ZONE 'UTC' AS TIMESTAMP) - max(r.submitted_at) AS time_since_review
+        FROM
+            pulls p
+                JOIN reviews r ON (r.owner, r.repo, r.pull_number) = (p.owner, p.repo, p.pull_number)
```

Comment:
nit: this could be a single line, same in other CTEs

Reply:
this looks consistent w/the rest of the query and the prettify from the SQL formatter?